lib/editor/result-view.js

```diff
@@ -83,18 +83,19 @@ export default class ResultView {
       if (ev.button === 1) {
         // middle mouse button
         this.model.remove()
-        this.focusEditor()
+        this.focusPrevious()
       } else if (document.getSelection().toString().length === 0) {
-        this.focusEditor()
+        this.focusPrevious()
       }
     })
 
     this.view.addEventListener('mousedown', ev => {
       this.blurEditor()
+      this.previousFocus = document.activeElement
```

Comment:
Does this [returned general `Element` object](https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement) always has `focus` property ?
I'm wondering if we need `if (this.previousFocus && this.previousFocus.focus)` check.

Reply:
Every `HTMLElement` has a `focus` method (https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/focus), but can't hurt to add the check.

Reply:
cool, I just wondered a returned can be `SVGElement` etc. that may not have `focus`.